kernel/config/definitions/kernel.yml

```diff
@@ -14,7 +14,7 @@
       - string:
           name: DISTROS
           description: "A list of distros to build for. Available options are: xenial, centos7, centos6, trusty, precise, wheezy, and jessie"
-          default: "centos7 precise xenial"
+          default: "centos7 xenial"
```

Comment:
Can we add trusty here as well? For repos that teuthology might use for testing we've been building for centos7, xenial and trusty.